Wemos_Modules/Chair/Chair.ino

```diff
@@ -0,0 +1,216 @@
+/*
+ * HHS PJSD 2021
+ *
+ * Groep 1
+ *
+ * Marko Estrada Rodriguez
+ * Wouter de Bruijn
+ * Casper Bastiaan-Net
+ * Maurice Tacx
+ * Steven Cowan
+ *
+ * Wemos I/O Server
+ *
+ * Module : Chair
+ *
+ * Wemos module connecting to a wifi network and creating a TCP server.
+ * Can receive TCP server commands to set or get data.
+ * Can use IO from WIB Board
+ *
+ * NOTE: using library ArduinoJson 6.18.5
+ */
+
+#include <Wire.h>
+#include <ArduinoJson.h>
+#include <ESP8266WiFi.h>
+#include <ArduinoWiFiServer.h>
+
+#define WIFI_SSID "G41C control node"
+#define WIFI_PASSWD "g41cstandsforgalc"
+#define PORT 8080
+#define ANALOG_IC_ADDR 0x36
+#define DIGITAL_IC_ADDR 0x38
+#define DIGITAL_IC_IN 0x00
+#define DIGITAL_IC_OUT 0x01
+
+/*const IPAddress local_IP(172, 16, 99, 100);
+const IPAddress gateway(172, 16, 99, 1);
+const IPAddress subnet(255, 255, 255, 0);*/
+
+void configureDigitalIC();
+void configureAnalogIC();
+
+unsigned int readDigitalInputs();
+void setDigitalOutput(const int pin, const bool state);
+
+unsigned int readAnalogInput(int ANALOG_CH);
+void handleConnections();
+void connectWifi();
+
+
+
+bool buttonChair = false;
+bool ledChair = false;
+bool vibrationMotor = false;
+unsigned int forceSensorChair = 0;
+byte digitalBuffer = 0;
+
+ArduinoWiFiServer server(PORT);
+
+void setup()
+{
+    // Start up Serial debug
+    Serial.begin(9600);
+
+    connectWifi();
+
+    Wire.begin();
+
+    delay(1000);
+
+
+    configureDigitalIC();
+    configureAnalogIC();
+
+    // Start TCP Server
+    server.begin();
+    Serial.println("TCP Server started listening...");
+
+   
+}
+
+void loop()
+{
+    // Handle clients sending request to the TCP server.
+    handleConnections();
+
+    int inputData = readDigitalInputs();
+    unsigned int analogInputData = readAnalogInput(0);
+
+    buttonChair = inputData & (1 << 0);
+    forceSensorChair = analogInputData;
+    
+   
+    setDigitalOutput((vibrationMotor << 5) | (ledChair << 4));
+
+    
+}
+
+void handleConnections()
+{
+    // Gets a client that is connected to the server and has data available for reading.
+    WiFiClient client = server.available();
+
+    if (client) // Check if client has send a message, otherwise this is false.
+    {
+        String s = client.readStringUntil('}'); // Read the incoming message. Delimited by a new line char.
+
+        StaticJsonDocument<100> jsonIn;
+        deserializeJson(jsonIn, s);
+        vibrationMotor = jsonIn["vM"];
+        ledChair = jsonIn["lChr"];
```

Comment:
Again the `Chr` or `C` or `Ch`  isn't needed here. This is all chair module specific.
There isn't any consistency in the names.